dask/tests/test_utils.py

```diff
@@ -894,3 +895,11 @@ def test_tmpfile_naming():
     with tmpfile(extension=".jpg") as fn:
         assert fn[-4:] == ".jpg"
         assert fn[-5] != "."
+
+
+def test_warn():
```

Comment:
It might be nice to test that this works both with and without `distributed` available.

A way you can simulate that is by temporarily setting `sys.modules['distributed']` to None. Here's an example of doing that in pytest with `monkeypatch`: https://stackoverflow.com/a/28361013/17100540. (You wouldn't need to make a fixture like that example does, though; you can just do it in the body of the test here.)